api/types/constants.go

```diff
@@ -554,6 +554,7 @@ var RequestableResourceKinds = []string{
 	KindDatabase,
 	KindApp,
 	KindWindowsDesktop,
+	KindKubePod,
```

Comment:
this will show up in the help for `tsh request search` but I don't think it will be supported there

Reply:
I have another PR that implements `tsh request search --kind=pod"